client/src/App.js

```diff
@@ -15,6 +15,7 @@ function App() {
   const [userData, setUserData] = useState({
     token: undefined,
     user: undefined,
+    polls: undefined,
```

Comment:
One thing to start thinking about is that you may notice a lot of user data (`user`, `polls`) is getting passed through components a lot now. We don't have to do this now, but you may also start taking a look at using the Context hook to store these states